subsys/profiler/profiler_nordic.c

```diff
@@ -109,10 +109,6 @@ static void send_system_description(void)
 			err = send_info_data(&end_line, 1);
 		}
 	}
-
-	if (!err) {
```

Comment:
I think this would break profiler. Host expects two end line characters to indicate end of the description. See https://github.com/nrfconnect/sdk-nrf/blob/17999765cd133f2816bdc0fe9007fac01b3d79fb/scripts/profiler/rtt2socket.py#L156-L173

Reply:
From host perspective, the second end of line is something like an empty description. This indicates that there are no more descriptions on device side.

Reply:
Ok I will leave it as this:

	`if (!err) {		
         send_info_data(&end_line, 1);
	}`